stacks/api/bucket/src/bucket-data.controller.ts

```diff
@@ -489,39 +333,24 @@ export class BucketDataController {
       );
     });
 
-    const paths = extractPropertyMap(schema.acl.write).map(path => path.split("."));
-
-    const relationMap = await createRelationMap({
-      properties: schema.properties,
-      paths,
-      resolve: (bucketId: string) => this.bs.findOne({_id: new ObjectId(bucketId)})
+    const currentDocument = await patchDocument(
+      schema,
+      {...patchedDocument, _id: documentId},
+      patch,
+      {req: req},
+      {
+        collection: bucketId => this.bds.children(bucketId),
+        schema: (bucketId: string) => this.bs.findOne({_id: new ObjectId(bucketId)})
+      },
+      {returnOriginal: false}
+    ).catch(error => {
+      throw error;
     });
```

Comment:
Same as above. I am not sure if this was intentional but this is the same as letting the original promise reject.

Reply:
removed